types/src/front/lib/dust_api.ts

```diff
@@ -717,6 +717,16 @@ export class DustAPI {
         text,
       }),
     });
+
+    if (!res.ok) {
```

Comment:
Actually we can have valid JSON with !res.ok

We want to try parse if fail fallback on your flow and otherwise return the error